src/gui/processing/models/qgsmodelgraphicsscene.cpp

```diff
@@ -224,6 +224,11 @@ void QgsModelGraphicsScene::createItems( QgsProcessingModelAlgorithm *model, Qgs
     const QMap<QString, QgsProcessingModelOutput> outputs = it.value().modelOutputs();
     QMap< QString, QgsModelComponentGraphicItem * > outputItems;
 
+    // offsets from algorithm item needed to correctly place output items
+    // which does not have valid position assigned (https://github.com/qgis/QGIS/issues/48132)
+    double output_offset_x = mChildAlgorithmItems[it.value().childId()]->component()->size().width();
```

Comment:
Would be good to retrieve `mChildAlgorithmItems[it.value().childId()]` once upfront and store in a variable for access elsewhere in this block (avoiding multiple map lookups)